src/answers-umd.js

```diff
@@ -428,6 +436,26 @@ class Answers {
       onFinally: config.onFinally || function () {}
     });
   }
+
+  /*
+   * Adds context as a parameter for the query API calls.
+   * @param {Object} context The context object passed in the API calls
+   */
+  setContext (context = {}) {
+    if (context && !isValidContext(JSON.stringify(context))) {
+      return;
+    }
+
+    this.core.persistentStorage.set(
+      StorageKeys.API_CONTEXT,
+      JSON.stringify(Object.assign({}, context)),
+      true
+    );
+    this.core.globalStorage.set(
+      StorageKeys.API_CONTEXT,
+      JSON.stringify(Object.assign({}, context))
```

Comment:
nit but could we use spread here?

Reply:
Ah didn't see this, but yes removed! Thank you!